.gitignore

```diff
@@ -0,0 +1,14 @@
+# Ignore Gradle project-specific cache directory
+.gradle
+
+# Ignore Gradle build output directory
+build
+
+.env
+app
```

Comment:
What does app/ folder consist of? Please list a few file names. Example, `src/example/example.java`.

I guess it should not be in .gitignore because app/ would consist of the core logic of the application.